app/scripts/directives/resources.js

```diff
@@ -1,6 +1,15 @@
 'use strict';
 
 angular.module('openshiftConsole')
+  .directive('initCount', function() {
```

Comment:
Do you mind making this an Angular component instead of directive? We've switched to components for new code.